kodilla-intro/src/main/java/User.java

```diff
@@ -0,0 +1,61 @@
+public class User {
+    private String[] name;
+    private int[] age;
+    private int size;
+
+    public User() {
+        this.name = new String[5];
+        this.age = new int[5];
+        this.size = 0;
+    }
+
+    public void addUser(String name, int age ) {
+        if (this.size == 5) {
+            return;
+        }
+        this.name[this.size] = name;
+        this.age[this.size] = age;
+        this.size++;
+    }
+
+    public double calcAveAge() {
+        int sum = 0;
+
+        for (int i = 0; i < this.size; i++) {
+            sum += this.age[i];
+        }
+
+        if (this.size > 0) {
```

Comment:
raczej najpierw sprawdzamy warunek, a potem liczymy :) Gdy size == 0 to bez sensu wykonujesz operacje z linii 22 - 31 :)